Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -498,25 +473,16 @@ protected async Task RequestHandler(IHttpRequest httpReq, string urlString, stri
                     return;
                 }
 
-                var operationName = httpReq.OperationName;
-
-                enableLog = EnableLogging(urlString, localPath);
```

Comment:
aren't logs going to be swamped by lots of static asset logs now?

Reply:
Everything is being logged at the debug level, except requests that take longer than 500ms.